mail-client/alpine/files/alpine-2.22-cc.patch

```diff
@@ -0,0 +1,20 @@
+--- ./imap/tools/Makefile	2020-04-19 23:45:13.080083548 +0200
++++ ./imap/tools/Makefile	2020-04-19 23:45:24.525083844 +0200
+@@ -21,7 +21,6 @@
+ #              Internet: MRC@CAC.Washington.EDU
+ 
+ 
+-CC=cc
+ RM=rm -f
+
+
+--- ./imap/src/osdep/unix/Makefile	2020-04-19 23:54:55.454098598 +0200
++++ ./imap/src/osdep/unix/Makefile	2020-04-19 23:54:55.454098598 +0200
+@@ -133,7 +133,6 @@
+ # Commands possibly overridden by the individual port
+ 
+ ARRC=ar rc
+-CC=cc
+ LN=ln -s
+ RANLIB=ranlib
```

Comment:
Since this patch is clearly about clang, you might as well patch out ar and ranlib, since llvm has its own tools for those.

Reply:
Done